api/app/Http/Controllers/ChildEntityController.php

```diff
@@ -224,7 +224,7 @@ public function deleteOne($id, $childId)
         $model = $this->findParentEntity($id);
         $childModel = $this->findOrFailChildEntity($childId, $model);
 
-        $childModel->delete();
+        $model->deleteChild($childModel, $this->relationName);
```

Comment:
@artstorm is this still needed? Looks like this requirement was refactored out right?